.github/workflows/test.yml

```
@@ -11,7 +11,9 @@ env:
   CTEST_OUTPUT_ON_FAILURE: 1
   GITHUB_HUNTER_USERNAME: ${{ secrets.HUNTER_USERNAME }}
   GITHUB_HUNTER_TOKEN: ${{ secrets.HUNTER_TOKEN }}
-  INDOCKER_IMAGE: soramitsu/kagome-dev:11
+  # Change all container: values to this variable once this feature is available
+  # https://github.community/t/how-to-set-and-access-a-workflow-variable/17335/6
+  INDOCKER_IMAGE: harrm/kagome-dev:1-minideb
```

I copied the image to company account:
```suggestion
  INDOCKER_IMAGE: soramitsu/kagome-dev:1-minideb
```